### PR TITLE
sink default to current directory

### LINKED ISSUE #2159: `fn sink` should write to current directory by default
Currently the command `fn sink` requires an argument `DIR` to point to the sink destination. To keep consistent with other commands, the argument should be optional and sink should write to PWD by default.

### PR DESCRIPTION
**Pull request by Shell32-Natsu**
close #2159 

`fn sink` will use `.` by default.

### CODE REVIEW

File: thirdparty/cmdconfig/commands/cmdsink/cmdsink.go [hunk 1]
```diff
@@ -35,7 +34,11 @@ type SinkRunner struct {
 }
 
 func (r *SinkRunner) runE(c *cobra.Command, args []string) error {
-	outputs := []kio.Writer{&kio.LocalPackageWriter{PackagePath: args[0]}}
+	dir := "."
```
Comment (frankfarzan) [inline thirdparty/cmdconfig/commands/cmdsink/cmdsink.go:37]: do you assume POSIX?

Comment (Shell32-Natsu) [reply thirdparty/cmdconfig/commands/cmdsink/cmdsink.go:37]: I believe this should also work on Windows. I notice that in other commands [some](https://github.com/GoogleContainerTools/kpt/blob/next/internal/cmdinit/cmdinit.go#L72) are using `pkg.CurDir` which is basically a `.` and [some](https://github.com/GoogleContainerTools/kpt/blob/next/internal/cmdliveinit/cmdliveinit.go#L84) are using `os.Getwd`.

Comment (frankfarzan) [reply thirdparty/cmdconfig/commands/cmdsink/cmdsink.go:37]: Should be consistent. Let's standardize on pkg.CurDir across the board.

Comment (Shell32-Natsu) [reply thirdparty/cmdconfig/commands/cmdsink/cmdsink.go]: done

**Review by droot — APPROVED** [review]

**Pull request merged**